plugins/star-rating/api/api.js

```diff
@@ -110,14 +452,87 @@ var plugin = {},
                         });
                         common.returnOutput(params, result);
                         return true;
-                        //common.returnMessage(params, 200, result);
                     }
                 }
             );
             return true;
         }
         return false;
     });
+    
+    plugins.register("/i/apps/create", function(ob){
+        var params = ob.params;
+        var appId = ob.appId;
+        common.db.collection('web_feedback_widgets_' + appId).insert({_id:"meta"},function(){});
+        common.db.collection('feedback' + appId).ensureIndex({"ts":1, "uid":1}, {unique:true}, function(){});
+    });
+
+    plugins.register("/i/apps/delete", function(ob){
+        var appId = ob.appId;
+        common.db.collection('web_feedback_widgets_' + appId).drop(function() {});
+        common.db.collection('feedback' + appId).drop(function() {});
+        if(common.drillDb)
+            common.drillDb.collection("drill_events" + crypto.createHash('sha1').update("[CLY]_star_rating" + appId).digest('hex')).drop(function() {});
+    });
+    
+    plugins.register("/i/apps/clear", function(ob){
+        var appId = ob.appId;
+        var ids = ob.ids;
+        common.db.collection('web_feedback_widgets_'+appId).remove({$and:[{'_id': {$regex: appId + ".*"}}, {'_id': {$nin:ids}}]},function(){});
```

Comment:
This completele does not look correct. For period deletion, we should not delete widgets at all.